README.adoc

```diff
@@ -14,6 +14,8 @@ Facilitate manipulation such as managing POM dependencies through CLI (command l
 
 == How to use it
 
```

Comment:
```suggestion

[TIP]
===============================
In order to make easier the plugin utilization we can register the groupId into the `settings.xml` like the snipped code below:

[source,xml]
----
<settings >
    <!-- snipped -->
    <pluginGroups>
      <pluginGroup>br.org.soujava</pluginGroup>
    </pluginGroups>
    <!-- snipped -->
</settings>
----

With that, we will be able to perform the plugin by using its prefix. 

So, instead of to perform the plugin by the following command:

[source, sh]
----
mvn br.org.soujava:pom-editor-maven-plugin:<GOAL>
----

We may perform it just providing its prefix like below:

[source, sh]
----
mvn pom-editor:<GOAL>
----

In the documentation, we'll assume that you have done this configuration.

===============================

```